housekeeper/cli/core.py

```diff
@@ -44,7 +45,8 @@ def base(
         raise click.Abort
     context.obj["database"] = db_path
     LOG.info("Use root path %s", root_path)
-    context.obj["store"] = Store(db_path, root_path)
+    initialise_database(db_path)
+    context.obj["store"] = Store(root=root_path)
```

Comment:
Given that the initialisation no longer occurs in the `Store` class constructor but instead here in the base CLI command, will it be configured correctly when used in `cg`?

Reply:
Nope, this is how it is initialised in `cg`:
```
    def __init__(self, config: dict) -> None:
        self._store = Store(config["housekeeper"]["database"], config["housekeeper"]["root"])
        self.root_dir: str = config["housekeeper"]["root"]
```

Reply:
https://github.com/Clinical-Genomics/cg/pull/2612